pkg/web/web.go

```diff
@@ -259,7 +274,8 @@ func New(ctx context.Context, opts ...Option) (*Server, error) {
 	return s, nil
 }
 
-func isZeros(buf []byte) bool {
+// IsZeros checks whether a byte array consists only of zeros.
+func IsZeros(buf []byte) bool {
```

Comment:
Not sure we should export such a small func only to use it in `cmd/ttn-lw-stack/commands/start.go`